effectful-th/src/Effectful/TH.hs

```diff
@@ -0,0 +1,238 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE TemplateHaskell #-}
+-- | Template Haskell utilities.
+module Effectful.TH
+  ( -- * Generate functions for dynamic effects
+    makeSendFunctions
+  , makeSendFunctionsWithOptions
+  , makeSendFunctionFor
+  , Options
+  , defaultOptions
+  , setMakeFunction
+  , setMakeFunctionFor
+  , setToFunctionName
+
+    -- * Re-exports
```

Comment:
Why the re-exports?

Reply:
I included them for slightly better ergonomics: This way one can just import `Effectful.TH` and is not confronted with a bunch of "... not in scope" error messages immediately.